src/data/ACTIONS/AST.js

```diff
@@ -62,8 +87,7 @@ export default {
 		icon: 'https://xivapi.com/i/003000/003123.png',
 		onGcd: true,
 		castTime: 3,
-		mpCostFactor: 16,
-		mpCost: 1920,
+		mpCost: 700,
```

Comment:
```suggestion
		mpCost: 600,
```
And its 5.01 ready https://eu.finalfantasyxiv.com/lodestone/topics/detail/80445702aebf22a9a5a32ff2fd1b69dcdaf6ac06